inc/Mailer.class.php

```diff
@@ -754,6 +755,13 @@ public function send() {
                 $headers .= MAILHEADER_EOL.trim($data['headers']);
             }
 
+            if(!function_exists('mail')){
+                $emsg = $lang['email_fail'] . $subject;
+                error_log($emsg);
+                msg($emsg,-1,__LINE__, __FILE__, MSG_MANAGERS_ONLY);
+                return false;
```

Comment:
Just read the event again, and we could have advise_after here (as `$success = false` will be passed to the event):

```suggestion
                $evt->advise_after();
                return false;
```

Reply:
Thanks for all the work.  I don't want to mess things up and am new to this kind of collaboration.  So I have a very basic question question: Do I commit both suggestions?  That is, if I do the first commit, that will remain in place if I commit the second?  Thanks again.

Reply:
Yes definitely, the later git commit will be based on the previous one which will keep both your changes. The reason that you need rebase before is about your base commit, not about the new commits.